### PR TITLE
Backport 4.2: Add Salt Bundle functionality extension with PIP modules (#1674)

### PR DESCRIPTION
**Pull request by keichwa**
# Description
* Add a note about using PIP with the Salt Bundle
* Update modules/client-configuration/pages/contact-methods-saltbundle.adoc
Co-authored-by: Alexander Graul <agraul@suse.com>
Co-authored-by: Karl Eichwalder <ke@suse.de>

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [ ] Manager-4.3
- [x] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by vzhestkov — CHANGES_REQUESTED** [review]
Thanks for backporting it.
But there is a mess in the changelog probably caused by cherry-picking...

File: .changelog [hunk 1]
```diff
@@ -4,8 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
-- Add missing part of the description to enable optional support of the Salt Bundle with Salt SSH
-- Added exact command to create the bootstrap repo for salt bundle and about how to disable salt-thin
```
Comment (agraul) [inline .changelog:8]: Are there entries removed on purpose?

**Review by agraul — COMMENTED** [review]
Thanks for the backport. I only have one question about the changelog, otherwise it LGTM.

**Comment by keichwa** [comment]
Thank you for spotting the changelog mess!  Fixed now.

**Review by vzhestkov — APPROVED** [review]
Looks good :+1: thanks for backporting it

**Pull request merged**